.ansible-lint

```diff
@@ -0,0 +1,15 @@
+---
+exclude_paths:
+# NOTE(luisd): AFAIK ansible-lint doesn't have a include files only and neither deos have
+#  a glob implementation, so we exclude all yaml files manually from the root directory :/
```

Comment:
I know this is just a note but since documentation is specially important in this project I thought that the wording should be corrected.

```suggestion
# NOTE(luisd): AFAIK ansible-lint has neither an "include files only" configuration nor a glob implementation, so we exclude all yaml files manually from the root directory :/
```